Caravan/about.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
+    <title>About</title>
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Alegreya+SC&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <main class="menu-page">
+        <h1 class="title">About Game</h1>
+        <article class="about-article">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+            incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+            exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+            dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+            mollit anim id est laborum.
+        </article>
+    </main>
+    <button class="back-button"></button>
```

Comment:
U sure it should not be a link?

Reply:
No, this is this button. It does not have a :hover effect, but has an :active. Anyway, It's to be replaced.

![image](https://user-images.githubusercontent.com/63360641/118377021-14b48080-b5d4-11eb-952d-108867b23a2d.png)

Appreciated for `9`. Pretty unexpected...